test/UpdatePropertyController.test.ts

```diff
@@ -0,0 +1,35 @@
+import { Database } from "../src/supabase/supabase";
+import { test_manualCreateProperty, test_manualDeleteProperty, bare_bones_testing_property } from "./TestUtil";
+import { updateProperty } from "../src/controllers/UpdatePropertyController";
+
+describe('updating a property', () => {
+    let property: Database['public']['Tables']['Properties']['Row'];
+
+    beforeAll(async () => {
+        // create temp property and room for tasks to correspond to
+        property = await test_manualCreateProperty();
+    })
+
+    afterAll(async () => {
+        // delete temp property cascades to delete room too
+        await test_manualDeleteProperty(property.property_id);
+    })
+
+    // test update property (remove room)
+    test('should update the property entry to match the new data', async () => {
+        let res = await updateProperty(property, '');
+
+        expect(res).toHaveProperty("room_ids", []);
+
+        await test_manualDeleteProperty(property.property_id);
```

Comment:
should this theoretically also test that the rooms are deleted, because that logic is part of the `updateProperty` method?

Reply:
whatever nvm, let's just get this done